libs/components/molecules/src/lib/landing-page-intro/landing-page-intro.component.html

```diff
@@ -0,0 +1,11 @@
+<div class="intro">
```

Comment:
Remove wrapping div and use grid for layout
Grid reference: https://css-tricks.com/snippets/css/complete-guide-grid/